grails-app/conf/application.yml

```diff
@@ -204,16 +204,17 @@ grails:
         # We now use the java.time classes instead (ex: Instant, LocalDate). Those fields define their own
         # ValueConverter classes and so don't use the formats defined here.
         dateFormats:
-            # Old date formats, don't use going forward! Kept first in the list to maintain backwards compatability.
+            # Date formats used by Excel and our old APIs. Kept first in the list to maintain backwards compatability.
             - 'MM/dd/yyyy HH:mm:ss XXX'
             - 'MM/dd/yyyy HH:mm:ss XX'
             - 'MM/dd/yyyy HH:mm:ss X'
             - 'MM/dd/yyyy HH:mm XXX'
             - 'MM/dd/yyyy HH:mm XX'
             - 'MM/dd/yyyy HH:mm X'
-            # Note that if this format is given, Grails will treat it as midnight in the local server timezone, which
-            # is ahead of UTC, can cause the date to actually be the next day. It can also cause inconsistencies across
-            # hosts/environments if they're configured for different local timezones. Avoid this format going forward!
+            # Note that Grails will treat this format as midnight in the local server timezone. For zones that are
+            # ahead of UTC, this can cause the date to actually become the next day. It can also cause
+            # inconsistencies across hosts/environments if they're configured for different local timezones.
+            # Avoid using this format going forward! Only kept to maintain backwards compatability.
             - 'MM/dd/yyyy'
```

Comment:
this problem will always be present so long as we use the Date class. Explicility calling
```
Date date = DateUtil.asDate("01/01/2000") 
```
will correctly default to UTC, but if we rely on Grails' databinding (ie have a controller method take in a Command object arg that has a Date field), Strings like "01/01/2000" will default to the local server time. This is why switching to Instant for new features is better